pyomo/util/tests/test_components.py

```diff
@@ -7,31 +7,32 @@
 #  rights in this software.
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
+
 from six.moves import zip_longest
 
 import pyutilib.th as unittest
 
-import pyomo.environ as pe
-import pyomo.kernel as pmo
```

Comment:
I think we should use `import pyomo.environ as pyo` and `import pyomo.kernel as pmo` to avoid confusion between AML and kernel components